server/controllers/stock/core.js

```diff
@@ -450,6 +456,11 @@ function getInventoryQuantityAndConsumption(params) {
       if (status) {
         return rows.filter(row => row.status === status);
       }
+
+      if(require_po) {
```

Comment:
It looks like `status and `require_po` are exclusive.  Can both options be set?  If so, these shouldn't `return rows.filter()`, they should do something like:
```js
var filteredRows = rows;

if (status) {
  filteredRows = filteredRows.filter(row => row.status === status);
}

if (require_po) {
  filteredRows = filteredRows.filter(row => row.S_Q > 0);
}

return filteredRows;
```